recipes/starshot_event_content_type/config/pathauto.pattern.events.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
 id: events
 label: Events
 type: 'canonical_entities:node'
-pattern: '/event/[node:field_date_time:start_date:html_year]/[node:title]'
+pattern: '/event/[node:field_date_time:value-custom:Y]/[node:title]'
```

Comment:
What does this resolve to? Will this be the start date or the end date?

Reply:
Yes, this will provide the start value formatted as a year. The tokens for Smart Date fields are a little different.